README.md

```diff
@@ -4,120 +4,60 @@ This repo has a prototype for a TANF Data Reporting system, where States,
-and merged in, circleci will do a full build/test/deploy to the cloud.gov account that you
-set up.
-
-## Connect to the database in cloud.gov
-
-First, you will need to install the [cf connect-to-service plugin](https://github.com/18F/cf-service-connect).
-
-After this is installed, you can get into the database in cloud.gov by doing
-`cf connect-to-service tanf tanf-db`.
-This will give you a postgres psql prompt, and you can do whatever queries you want.
-The `upload_*` tables are what contain all the data.
-
-If you would like to connect a GUI client to the database, then you can run
-`cf connect-to-service --no-client tanf tanf-db`, and it will supply you with the host/port,
-username/password, and database name for you to configure your client to use. 
-
-Note:
-the credentials will only work and the database will only be accessible while
-you have your authenticated `connect-to-service` session going.
+OFA has partnered with 18F to conduct the initial research, scoping, prototyping,
+and strategy for the new TDRS. 18F has laid the groundwork for a modern and sustainable
+software development process and removed as many barriers as possible for the incoming
+contractor team. This information may highlight important considerations for future
+development team, and should not constrain vendor’s technical approach submissions.
+
+The current TDRS is so old, fragile, and difficult to access that 18f only leveraged
+the existing data format that grantees use to submit their data to TDRS and did not
+develop any direct access to the system. This data format is not extensible, not
+human-readable, and does not have adequate documentation. However, the data format
+is standardized and does not immediately require the grantees to change their data
+generation processes. Both OFA staff and grantees would like a user interface for
+entering data in the new  system, but developing this will require more user research.
+
+This TDRS prototype was built to test assumptions about the primary goal of this project.
+The prototype allows users to upload data in the TANF data
+reporting format and view the resulting data. It conducts limited data validation of the
+submissions.  It is a Python/Django app with Login.gov integration, and running in Cloud.gov.
+The OFA team uses SQL with their existing analysis tools, so 18F developed the prototype
+with a Postgres SQL database. The contractor may extend, extract useful parts, or replace
+the prototype application entirely.
+
+The prototype provided an opportunity to develop an application environment. 18F and OFA
+have been working together to get core technical infrastructure approved and set up so
+when the contractor joins the team they can immediately begin to contribute code and have
+it automatically roll out to their development environment.
+
+The application environment is roughly: 
+
+![diagram of prototype apps and services](TANF_Infrastructure.png)
```

Comment:
Looks like the `.png` referenced here was removed.

Reply:
No, it's there.